weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -605,8 +608,22 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
   ) : (
     <Styles.Main
       data-weavepath={props.pathEl ?? 'root'}
+      onMouseDown={event => {
+        event.stopPropagation();
+        setLastMouseDownCoords({x: event.clientX, y: event.clientY});
+      }}
       onClick={event => {
-        if (isMain(fullPath) || isInsideMain(fullPath, 1)) {
+        if (isInsideMain(fullPath, 1)) {
+          setSelectedPanel(fullPath);
+          event.stopPropagation();
```

Comment:
maybe just toss event.stopPropagation() outside the if since both use it

Reply:
Done, also changed it from a useState to useRef